test/LibraryManager.Test/Providers/Unpkg/UnpkgCatalogTest.cs

```diff
@@ -1,110 +1,100 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Collections.Generic;
-using System.IO;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Microsoft.Web.LibraryManager.Contracts;
 using Microsoft.Web.LibraryManager.LibraryNaming;
-using Microsoft.Web.LibraryManager.Mocks;
 using Microsoft.Web.LibraryManager.Providers.Unpkg;
 
 namespace Microsoft.Web.LibraryManager.Test.Providers.Unpkg
 {
     [TestClass]
     public class UnpkgCatalogTest
     {
-        private ILibraryCatalog _catalog;
-        private IProvider _provider;
-
-        [TestInitialize]
```

Comment:
Setup was required before this change because we had static singleton's providing state (namely the Dependencies and LibraryIdToNameAndVersionConverter), and that state was persisted across test classes.  So, for example, this class initialized the Dependencies with just the Unpkg provider.  The JsDelivr tests need to then re-initialize with that provider; the same for Cdnjs, etc.  This was an issue when running tests *sequentially*, parallel would have been even worse.

All this overhead was just to set up the _dependencies.GetProvider() call, which we don't even need if we're testing the catalog.  And by further reducing the dependency on the provider, the catalog can now be tested without any of this overhead (by not needing the provider, we don't need the Dependencies and HostInteraction to also be pre-configured).